platform-workflow-module/src/main/java/com/softicar/platform/workflow/module/WorkflowI18n.java

```diff
@@ -37,8 +39,8 @@
 		.de("Automatische Übergänge dürfen keine Rollen definieren.");
 	I18n0 CLOSED = new I18n0("Closed")//
 		.de("Geschlossen");
-	I18n0 CONFIGURE_WORKFLOW_SUBSTITUTE = new I18n0("Configure Workflow Substitute")//
-		.de("Arbeitsablauf-Vertretung konfigurieren");
+	I18n0 CONFIGURE_SETTINGS = new I18n0("Configure settings")//
```

Comment:
"Configure settings" seems overly complicated and unnecessarily long to me.
I think we should just go with "Settings" or "Preferences" - that's how such buttons or menu items are typically labelled, in pretty much any application out there.
We should also rename "AGWorkflowUserConfiguration" to "AGWorkflowUser[Settings|Preferences]", respectively.